siliconcompiler/core.py

```diff
@@ -3798,7 +3800,7 @@ def _runtask(self, step, index, status):
         self.set('arg', 'step', None, clobber=True)
         self.set('arg', 'index', None, clobber=True)
         os.makedirs('inputs', exist_ok=True)
-        #self.write_manifest(f'inputs/{design}.pkg.json')
+        self.write_manifest(f'inputs/{design}.pkg.json')
```

Comment:
Can we remove this? Is it used?

Reply:
I don't think that anything in SC relies on it, but adding it back may be a good idea.

Pros:
* Saves a copy of the manifest to each task's `inputs/` directory before each task is run; could help with debugging.
* There is a comment indicating that it should be done.

Cons:
* May add a very small amount of runtime to each task, as the manifest is processed and written to disk.

Reply:
I would vote for keeping it in. I've wished for it to be there several times